.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL=mongodb://localhost/quizzes 
```

Comment:
Generally, `.env` files are not committed. `dotenv` also mentions this in their readme: https://github.com/motdotla/dotenv#should-i-commit-my-env-file

However, for now we can keep it committed since it will make it easier for everyone else in the team to get up and running, and when we get to deploying the app to a server we can remove the `.env` file